packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -640,7 +641,7 @@ export class Function extends FunctionBase {
 
   protected readonly canCreatePermissions = true;
 
-  private readonly layers: ILayerVersion[] = [];
+  public readonly layers: ILayerVersion[] = [];
```

Comment:
The alternative to making this public is to send the layers directly into the `calculateFunctionHash()` like: `calculateFunctionHash(fn, fn.layers)`. That seems awkward to me so I chose to make this public.

Reply:
`calculateFunctionHash` is a private function, so I don't really care about its signature. I care more about making this public, which this change does... so I'd rather you didn't.

Reply:
Marked @internal instead. I need this to be publicly accessible for the aspect as well, and if my understanding of `@internal` is correct, this makes the most sense.